src/pages/about-popover/about-popover.ts

```diff
@@ -1,16 +1,37 @@
 import { Component } from '@angular/core';
 
-import { ViewController } from 'ionic-angular';
+import { ViewController, NavController, App, ModalController } from 'ionic-angular';
+
+import { Support } from '../support/support';
 
 
 @Component({
-  templateUrl: 'about-popover.html'
+  template: `
+    <ion-list>
+      <button ion-item (click)="close('http://ionicframework.com/docs/v2/getting-started/')">Learn Ionic</button>
+      <button ion-item (click)="close('http://ionicframework.com/docs/v2/')">Documentation</button>
+      <button ion-item (click)="close('http://showcase.ionicframework.com/')">Showcase</button>
+      <button ion-item (click)="close('https://github.com/driftyco/ionic')">GitHub Repo</button>
+      <button ion-item (click)="support()">Support</button>
+    </ion-list>
+  `
 })
 export class PopoverPage {
 
-  constructor(public viewCtrl: ViewController) { }
+  constructor(
+    public viewCtrl: ViewController,
+    public navCtrl: NavController,
+    public app: App,
+    public modalCtrl: ModalController
+  ) { }
+
+  support() {
```

Comment:
We should add Support as an Account link. I didn't even know this page existed until I was going through the code.